### DISCUSSION TITLE
please check in the g4 of the mermaid which can help developers to show flowgraph as so on

### DISCUSSION THREAD

**Discussion by stride83** [antlr/grammars-v4#3938] [Q&A]
the mermaid is so useful for it developers , at the same time , people can use mermaid to driven the program running as the mermaid show . so need the mermaid grammar g4 file .
thanks

**Comment by stride83** [comment]
# first step: to develop 【flowgraph】 grammar  
# second step: to develop 【stateDiagram-v2】 grammar 
...

**Reply by mickroll** [reply]
have a look at https://github.com/aresstack/mermaid-java/tree/main/src/main/antlr4/com/aresstack/mermaid/parser, license is MIT
